src/main/api/2/studio-api-2.yaml

```diff
@@ -4323,6 +5253,291 @@ components:
+        path:
+          type: string
+          description: path where content item is stored inside repository (with filename)
+        previewUrl:
+          type: string
+          description: url for previewing content item
+        systemType:
+          type: string
+          description: one of predifined type values (page, component, asset, level descriptor, folder ...)
+        mimeType:
+          type: string
+          description: mime-type of content item file
+        state:
+          type: integer
+          format: int64
+          description: >-
+            bitmap of states
+              * `BIT ORDER (little-endian)`: 0, `FLAG`: New, `DESCRIPTION`: The item is new and never been published
+              * `BIT ORDER (little-endian)`: 1: `FLAG`: Modified, `DESCRIPTION`: The item has been published in the past and has been modified in the sandbox
+              * `BIT ORDER (little-endian)`: 2: `FLAG`: Deleted, `DESCRIPTION`: The item is deleted
+              * `BIT ORDER (little-endian)`: 3: `FLAG`: User Locked, `DESCRIPTION`: The item is locked by a user
+              * `BIT ORDER (little-endian)`: 4, `FLAG`: System Processing, `DESCRIPTION`: The item is locked by the system for processing
+              * `BIT ORDER (little-endian)`: 5, `FLAG`: In Workflow, `DESCRIPTION`: The item has been submitted to the workflow engine
+              * `BIT ORDER (little-endian)`: 6, `FLAG`: Scheduled, `DESCRIPTION`: The item has been scheduled for publishing
+              * `BIT ORDER (little-endian)`: 7, `FLAG`: Staged, `DESCRIPTION`: The item has been pushed to the staging environment
+              * `BIT ORDER (little-endian)`: 8, `FLAG`: Live, `DESCRIPTION`: The item has been pushed to the live environment
+              * `BIT ORDER (little-endian)`: 9, `FLAG`: Translation Up To Date, `DESCRIPTION`: This item is a translation of another and it’s up to date
+              * `BIT ORDER (little-endian)`: 10, `FLAG`: Translation Pending, `DESCRIPTION`: This item is a translation of another and it’s pending translation
+              * `BIT ORDER (little-endian)`: 11, `FLAG`: Translation in Progress, `DESCRIPTION`: This item is a translation of another and is currently being translated
+        lockOwner:
+          type: string
+          description: username of user that locked item for editing
+        disabled:
+          type: boolean
+          description: true if item is disabled for rendering in delivery, otherwise false
+        localeCode:
+          type: string
+          description: locale of content item
+        translationSourceId:
+          type: string
+          description: ID of content item that is original for translation that current item represents
+      required:
+        - id
+        - label
+        - contentTypeId
+        - path
+        - previewUrl
+        - systemType
+        - mimeType
+        - state
+        - lockOwner
+        - disabled
+        - localeCode
+
+    DashboardItem:
```

Comment:
Why do we need this one?

Reply:
That will be an item object for displaying on the dashboard. In general use case, it is not a detailed item, but it is different from sandbox item